integrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Integrations
+permalink: /integrations/
+redirect_from:
+  - /source-code/
+  - /downloads/
+---
+
+
+<a name="packages"></a>
```

Comment:
`<a name="packages"></a>`... what's this supposed to do?

Reply:
Nothing sorry

Reply:
Ah, it creates linkable anchors for the list. But I guess it should wrap the headings then, shouldn't it?

Reply:
I delete it on the new commit